frontend/app/controllers/Event.scala

```diff
@@ -66,7 +66,7 @@ trait Event extends Controller with ActivityTracking {
    * Note that Composer will index this data, which is in turn indexed by CAPI.
    * (eg. updates to event details will not be reflected post-embed)
    */
-  def embedData(slug: String, callback: String) = CachedAction { implicit request =>
+  def embedData(slug: String) = Cors.andThen(CachedAction) { implicit request =>
```

Comment:
FYI. When https://github.com/guardian/membership-frontend/pull/477 goes out we can use `AjaxCachedAction` as a shorthand for this (should probably be named `CorsCachedAction` but there was already a `AjaxNoCacheAction` which did `Cors + NoCache`…)